packages/scandipwa/src/store/Cart/Cart.dispatcher.js

```diff
@@ -199,8 +199,12 @@ export class CartDispatcher {
 
             this._updateCartData(cartData, dispatch);
             dispatch(showNotification('success', __('Coupon was applied!')));
+
+            return true;
         } catch (error) {
             dispatch(showNotification('error', getErrorMessage(error)));
+
+            return null;
```

Comment:
If a method returns a boolean, it's better to be consistent and return boolean in all code branches.
Please return false instead of null if possible.